test/Spec.hs

```diff
@@ -151,6 +152,7 @@ main = hspec $ do
     it "pure absorption on commit" $ property $ absorptionPureCommit (undefined :: TacticTest Int)
     it "empty identity on commit" $ property $ emptyIdentityCommit (undefined :: TacticTest Int)
     it "failure identity on commit" $ property $ emptyIdentityCommit (undefined :: TacticTest Int)
+    -- it "constant peek" $ property $ peekConst (undefined :: TacticTest ())
```

Comment:
Commented out on purpose?

Reply:
Yeah, test still fails for reasons that I don't totally understand

Reply:
Ok they fail because `peek` is broken who woulda thunk. reify machinery does this right.

Reply:
wow, tests hey?